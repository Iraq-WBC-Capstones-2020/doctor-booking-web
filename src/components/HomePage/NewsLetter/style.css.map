{
    "version": 3,
    "mappings": "AAAA,AAAA,IAAI,CAAC;EACH,WAAW,EAAE,MAAM;CACpB;;AAED,AAAA,WAAW,CAAC;EACV,MAAM,EAAE,eAAe;CACxB;;AAED,AAAA,WAAW,CAAC;EACV,MAAM,EAAE,KAAK;EACb,gBAAgB,EAAE,OAAO;EACzB,WAAW,EAAE,IAAI;EACjB,WAAW,EAAE,MAAM;EACnB,aAAa,EAAE,IAAI;CAapB;;AAlBD,AAOE,WAPS,CAOT,EAAE,CAAC;EACD,WAAW,EAAE,IAAI;EACjB,cAAc,EAAE,MAAM;EACtB,KAAK,EAAE,OAAO;CACf;;AAXH,AAaE,WAbS,CAaT,CAAC,CAAC;EACA,SAAS,EAAE,IAAI;EACf,WAAW,EAAE,IAAI;EACjB,KAAK,EAAE,OAAO;CACf;;AAGH,AAAA,WAAW,CAAC;EACV,MAAM,EAAE,MAAM;CACf",
    "sources": [
        "style.scss"
    ],
    "names": [],
    "file": "style.css"
}