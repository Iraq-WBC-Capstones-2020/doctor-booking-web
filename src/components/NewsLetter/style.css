body {
  font-family: Roboto;
}

.newsContainer {
  background-color: #f5f9fd;
}

.inputFiled {
  height: 48px !important;
}

.newsLetter {
  height: 277px;
  padding-top: 57px;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.newsLetter h1 {
  line-height: 48px;
  letter-spacing: -0.4px;
  color: #3f3b3b;
}

.newsLetter p {
  font-size: 18px;
  line-height: 24px;
  color: #3f3b3b;
}

.InputGroup {
  margin: 0 auto;
}
/*# sourceMappingURL=style.css.map */
