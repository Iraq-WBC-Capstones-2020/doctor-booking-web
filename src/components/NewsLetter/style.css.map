{
    "version": 3,
    "mappings": "AAAA,AAAA,WAAW,CAAC;EACR,WAAW,EAAE,MAAM;CACtB;;AAED,AAAA,WAAW,CAAA;EACP,MAAM,EAAE,IAAI;EACZ,KAAK,EAAE,gBAAgB;CAC1B",
    "sources": [
        "style.scss"
    ],
    "names": [],
    "file": "style.css"
}