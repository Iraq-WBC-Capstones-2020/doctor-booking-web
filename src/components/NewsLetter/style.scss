body{
    font-family: Roboto;
}

.inputFiled{
    height: 48px !important;
}

.NewsLetter{
    height: 277px;
    background-color: #D8D8D8;
    padding-top: 57px;
    align-items: center;
    margin-bottom: 48px;

    h1{
        line-height: 48px;
        letter-spacing: -0.4px;
        color: #3F3B3B;
    }

    p{
        font-size: 18px;
        line-height: 24px;
        color: #3F3B3B;
    }
}

.InputGroup{
    margin: 0 auto;
}