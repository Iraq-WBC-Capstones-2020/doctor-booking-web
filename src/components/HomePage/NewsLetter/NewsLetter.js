import React from 'react';
import { Container, Col, Row, InputGroup, FormControl } from 'react-bootstrap';
import ButtonImg from './images/Slice1.png';
import './style.css';
import { useTranslation } from 'react-i18next';
function NewsLetter() {
  const { t, i18n } = useTranslation(['common', 'validation']);

  return (
    <Container className="newsLetter">
      <div className="theHead">
        <Row className="align-items-center justify-content-center">
          <h1>{t('validation:homePage.newsLetter.headLine')}</h1>
        </Row>
        <Row className="align-items-center justify-content-center">
          <p>{t('validation:homePage.newsLetter.paragraph')}</p>
        </Row>
      </div>
      <div className="email">
        <Row className="align-items-center justify-content-center">
          <Col sm={4}>
            <InputGroup className="mb-3">
              <FormControl
                className="inputFiled"
                placeholder={t('common:email')}
                aria-label="Recipient's username"
                aria-describedby="basic-addon2"
              />
              <InputGroup.Append>
                <input
                  type="image"
                  src={ButtonImg}
                  alt="Submit"
                  width="48"
                  height="48"
                />
              </InputGroup.Append>
            </InputGroup>
          </Col>
        </Row>
      </div>
    </Container>
  );
}

export default NewsLetter;
