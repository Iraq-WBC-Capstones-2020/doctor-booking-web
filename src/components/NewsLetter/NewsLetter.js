import React from 'react';
import { Container, Col, Row, InputGroup, FormControl } from 'react-bootstrap';
import ButtonImg from './images/Slice 1.png';
import './style.css';
function NewsLetter() {
  return (
    <Container className="NewsLetter">
      <div className="theHead">
        <Row className="align-items-center justify-content-center">
          <h1>Never miss an update</h1>
        </Row>
        <Row className="align-items-center justify-content-center">
          <p>Signup to get any update and serves we add </p>
        </Row>
      </div>
      <div className="Email">
        <Row className="align-items-center justify-content-center">
          <Col sm={4}>
            <InputGroup className="mb-3">
              <FormControl
                className="inputFiled"
                placeholder="Recipient's username"
                aria-label="Recipient's username"
                aria-describedby="basic-addon2"
              />
              <InputGroup.Append>
                <input
                  type="image"
                  src={ButtonImg}
                  alt="Submit"
                  width="48"
                  height="48"
                />
              </InputGroup.Append>
            </InputGroup>
          </Col>
        </Row>
      </div>
    </Container>
  );
}

export default NewsLetter;
