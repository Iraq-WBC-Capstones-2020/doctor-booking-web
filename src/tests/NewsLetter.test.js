import React from 'react';
import { render } from '@testing-library/react';
import NewsLetter from '../components/NewsLetter/NewsLetter';
import renderer from 'react-test-renderer';
it('render1', () => {
  render(<NewsLetter />);
});

it('renders correctly', () => {
  const tree = renderer.create(<NewsLetter />).toJSON();
  expect(tree).toMatchSnapshot();
});
