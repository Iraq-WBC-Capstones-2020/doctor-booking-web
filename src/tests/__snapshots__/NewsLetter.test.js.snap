// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="newsContainer"
>
  <div
    className="newsLetter container"
  >
    <div
      className="theHead"
    >
      <div
        className="align-items-center justify-content-center row"
      >
        <h1>
          Never miss an update
        </h1>
      </div>
      <div
        className="align-items-center justify-content-center row"
      >
        <p>
          Signup to get any update and serves we add 
        </p>
      </div>
    </div>
    <div
      className="email"
    >
      <div
        className="align-items-center justify-content-center row"
      >
        <div
          className="col-sm-4"
        >
          <div
            className="mb-3 input-group"
          >
            <input
              aria-describedby="basic-addon2"
              aria-label="Recipient's username"
              className="inputFiled form-control"
              placeholder="Recipient's username"
            />
            <div
              className="input-group-append"
            >
              <input
                alt="Submit"
                height="48"
                src="Slice1.png"
                type="image"
                width="48"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
